tron/mcp.py

```diff
@@ -156,12 +144,9 @@ def restore_state(self):
         to the configured Jobs and Services.
         """
         self.event_recorder.notice('restoring')
-        job_states, service_states = self.state_watcher.restore(
-            self.jobs.get_names(), self.services.get_names(),
-        )
+        job_states = self.state_watcher.restore(self.jobs.get_names())
```

Comment:
This is probably the this that worries me the most: can we survive the transition from services to no-services (with regard to state-loading)?

Probably.